lib/Digitick/Sepa/TransferFile/Facade/BaseCustomerTransferFileFacade.php

```diff
@@ -64,4 +64,20 @@ public function asXML()
 
         return $this->domBuilder->asXml();
     }
+
+    /**
+     * @return DateTime  
+    */
+    public function HandleDueDate(array $paymentInformation)
```

Comment:
Methods start lowerCase

Reply:
Sorry, I should have read more code and check for style. Thanks for the welcome :-D

Reply:
And the advices. I was more in the subversion team, and I do not understand the whole Github system yet.